lib/components/InputBox/BaseInputBox.tsx

```diff
@@ -12,9 +12,11 @@ import {
   StyledInput,
   HintIconWrapper,
   HelperIcon,
+  ShowHidePwdBtn,
```

Comment:
please rename this to Password instead of PWD. pwd means something else and makes the code confusing.